arraymancer.nimble

```diff
@@ -1,11 +1,11 @@
 ### Package
-version       = "0.6.1"
+version       = "0.6.2"
 author        = "Mamy André-Ratsimbazafy"
 description   = "A n-dimensional tensor (ndarray) library"
 license       = "Apache License 2.0"
 
 ### Dependencies
-requires "nim >= 1.0.0",
+requires "nim >= 1.4.0",
```

Comment:
We could in principle define `IndexDefect` if we're running on an older Nim. I'm not sure if we need to throw out Nim 1.2 just because of Error vs Defect.

Reply:
We absolutly can do that. I just try to avoid boilerplate code whenever I can, but if the 1.4.0 requirement is too harsh, then your suggesiton is better.

Reply:
Let's wait for mratsim's input here.